localcoinbot.py

```diff
@@ -2,15 +2,19 @@
 
 from telegram.ext import Updater, CommandHandler, MessageHandler, Filters, CallbackQueryHandler, CallbackContext
 from telegram import ParseMode, InlineKeyboardButton, InlineKeyboardMarkup, Update, ChatPermissions
-
-import requests
+from pycoingecko import CoinGeckoAPI
```

Comment:
Good choice, but let's double check rate limits

Reply:
Rate limit for the CoinGecko API is 10 calls each second per Internet Protocol (“IP”) address - should be plenty